docs/examples/kube2iam.rst

```diff
@@ -0,0 +1,242 @@
+
+    terraform {}
+
+    provider "aws" {
+        region = "${var.aws_region}"
+    }
+
+    variable "aws_region" {
+        description = "AWS Region you want to deploy it in"
+        default     = "eu-west-1"
+    }
+
+    variable "cluster_name" {
+        description = "Name of the cluster"
+    }
+
+    data "aws_caller_identity" "current" {}
+
+    resource "aws_iam_policy" "kube2iam" {
+        name        = "kube2iam_assumeRole_policy_${var.cluster_name}"
+        path        = "/"
+        description = "Kube2IAM role policy for ${var.cluster_name}"
+
+        policy = "${data.aws_iam_policy_document.kube2iam.json}"
+    }
+
+    data "aws_iam_policy_document" "kube2iam" {
+        statement {
+            sid = "1"
+
+            actions = [
+            "sts:AssumeRole",
+            ]
+
+            effect = "Allow"
+
+            resources = [
+            "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/kube2iam_${var.cluster_name}/*",
+            ]
+        }
+    }
+
+    output "kube2iam_arn" {
+        value = "${aws_iam_policy.kube2iam.arn}"
+    }
+
+    output "kube2iam_path" {
+        value = "${aws_iam_policy.kube2iam.path}"
+    }
+
+
+You can run the Terraform project the following way:
+
+.. code-block:: bash
+
+    terraform init
+    terraform apply -var cluster_name=example -var region=eu-west-1
+
+Attach instance policy
+++++++++++++++++++++++
+
+Add the created IAM policy ARN to your tarmak config. You can do this by
+adding `additional IAM policies <https://docs.tarmak.io/user-guide.html#additional-iam-policies>`_.
+
+Deploy kube2iam
++++++++++++++++
+
+With `HELM <https://www.helm.sh/>`_ it is easy to deploy kube2iam with 
+the correct settings.
+
+You can deploy it with the following command:
+
+.. code-block:: bash
+
+    helm upgrade kube2iam stable/kube2iam \
+    --install \
+    --version 0.9.0 \
+    --namespace kube-system \
+    --set=extraArgs.host-ip=127.0.0.1 \
+    --set=extraArgs.log-format=json \
+    --set=updateStrategy=RollingUpdate \
+    --set=rbac.create=true \
+    --set=host.iptables=false
+
+
+We set ``iptables`` to false and ``host-ip`` to 127.0.0.1 as Tarmak already creates
+the iptables rule and forward it to ``127.0.0.1:8181``.
+Specific kube2iam options can be found in the `documentation <https://github.com/jtblin/kube2iam>`_ of kube2iam.
+
+Usage
+~~~~~
+
+Now that kube2IAM is installed on your system, you can start creating roles
+and policies to give your pods access to AWS resources.
+
+An example creation of an IAM policy and role:
+
+.. code-block:: none
+
+    terraform {}
+
+    provider "aws" {
+        region = "${var.aws_region}"
+    }
+
+    variable "aws_region" {
+        description = "AWS Region you want to deploy it in"
+        default     = "eu-west-1"
+    }
+
+    variable "cluster_name" {
+        description = "Name of the cluster"
+    }
+
+    variable "instance_iam_role_arn" {
+        description = "ARN of the instance IAM role
```

Comment:
```
$ terraform init
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Error parsing /Users/luke/Desktop/nginx/test.tf: At 19:48: literal not terminated
```